packages/Molecules/WhatsNext/index.vue

```diff
@@ -6,7 +6,7 @@
     <div class="rpl-whats-next__row">
       <ul class="rpl-whats-next__items" v-if="links">
         <li class="rpl-whats-next__item" v-for="(item, index) of links" :key="index">
-          <rpl-text-link :url="item.url" :text="item.text" :underline="true" size="small" />
+          <rpl-text-link :url="item.url" :text="item.text" icon-symbol="arrow_right_primary" icon-size="small" :underline="true" size="small" />
```

Comment:
Same `icon-size` issue as above.

Reply:
ah yes, must have got mixed up with the size prop which takes 'small'. Might be a good idea to standardize constant units. I've updated the PR.